pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -578,21 +574,34 @@ public CompletableFuture<Void> closeAsync() {
         }
     }
 
+    private void close(AutoCloseable closeable, String resource, ScheduledExecutorService shutdownExecutor) {
+        final CompletableFuture<Void> future = new CompletableFuture<>();
+        shutdownExecutor.execute(() -> {
+            try {
+                closeable.close();
+                future.complete(null);
+            } catch (Exception e) {
+                LOG.warn("Failed to close {}", resource, e.getMessage());
```

Comment:
The `e.getMessage()` may not be printed. I think it should be 
```
LOG.warn("Failed to close {}", resource, e);
```
or
```
LOG.warn("Failed to close {} with exception {}", resource, e.getMessage());
```

Reply:
fixed.